flask_webapp/app.py

```diff
@@ -1,15 +1,24 @@
 from flask import Flask, jsonify, request, redirect, render_template
+from camera import VideoCamera
+import cv2
 import sys
 
-if sys.platform == 'win32':
-    print("Running on Windows OS. This is not supported yet.")
-    exit()
+# if sys.platform == 'win32':
+#     print("Running on Windows OS. This is not supported yet.")
+#     exit()
 
 from src.device_list import BtDevContainer
 Container = BtDevContainer()
 
 app = Flask(__name__)
 
+def gen_frames(camera):
```

Comment:
This will need to be encased in a thread later and we'll need a flag to check when it's active (maybe store the camera state in the db and check it upon the browser loading the view)